virtue is not needed to run tests.
We have no virtue packaged yet.

--- jsonschema-4.25.1/pyproject.toml.orig
+++ jsonschema-4.25.1/pyproject.toml
@@ -80,7 +80,7 @@
 Source = "https://github.com/python-jsonschema/jsonschema"
 
 [dependency-groups]
-test = ["virtue"]
+test = []
 
 [tool.hatch.metadata.hooks.fancy-pypi-readme]
 content-type = "text/x-rst"
